packages/react-components/src/molecules/ModalEditHeader.tsx

```diff
@@ -36,7 +36,7 @@ const ModalEditHeader: React.FC<ModalEditHeaderProps> = ({
 }) => (
   <div css={headerStyles}>
     <div css={controlsContainerStyles}>
-      <Button onClick={onSave} primary small>
+      <Button onClick={onSave} primary small submit>
```

Comment:
This should be default for `primary` buttons already. Not saying it hurts to have it, just so you're aware

Reply:
Removed